### DISCUSSION TITLE
Left arrow key pressing itself

### DISCUSSION THREAD

**Discussion by carolalbs** [OpenRCT2/OpenRCT2#23116] [Q&A]
Hi everyone, I've been playing OpenRCT2 on my Mac for a while. Right now, it's just impossible to play because the left arrow key on the keyboard mysteriously starts pressing itself as soon as I open the game. Has anyone else had this problem?

**Comment by ghost** [comment]
I found a solution! You can remove the left arrow key in the settings so it doesn't scroll the map.

I don't speak english, but you can find the button below

![afbeelding](https://github.com/user-attachments/assets/48163da1-cbb1-4535-b10a-032d797a8a77)

now remove left key here

![afbeelding](https://github.com/user-attachments/assets/d487e475-e03d-4113-a560-4bd535cc1692)
